test/Microsoft.AspNetCore.Http.Connections.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -1744,12 +1839,18 @@ private static DefaultHttpContext MakeRequest(string path, ConnectionContext con
             return context;
         }
 
-        private static void SetTransport(HttpContext context, HttpTransportType transportType)
+        private static void SetTransport(HttpContext context, HttpTransportType transportType, CancellationToken terminateTransport = default)
         {
+            context.RequestAborted = terminateTransport;
             switch (transportType)
             {
                 case HttpTransportType.WebSockets:
-                    context.Features.Set<IHttpWebSocketFeature>(new TestWebSocketConnectionFeature());
+                    var feature = new TestWebSocketConnectionFeature();
+                    if (terminateTransport.CanBeCanceled)
```

Comment:
Eerr, why?

Reply:
I was probably going overboard trying to shut down the mock connection. I can yank it.